### DISCUSSION TITLE
Emulate other Jetson Orin module on AGX Orin

### DISCUSSION THREAD

**Discussion by apbr** [OE4T/meta-tegra#1118] [Q&A]
According to Nvidia it is possible to emulate the other Jetson Orin modules on the Jetson Orin AGX DevKit. [[1]](https://developer.nvidia.com/blog/develop-for-all-six-nvidia-jetson-orin-modules-with-the-power-of-one-developer-kit/?mkt_tok=MTU2LU9GTi03NDIAAAGHBQ5CeUcuw5baKWHoqYyWY8-w5QNefSGTbWrpamKDR4tWMEiDrbXmxl3ei9HqXKd_KTT_878SjXdrTbaSwgw2t_G_rsLDlt6UjRFN-Mkylp6NzWS3fA)

Is this already possible when using meta-tegra and Yocto?

Thank you for any help.

**Comment by madisongh** [comment]
We don't have machine configs in the layer for those alternative configurations.  It should be possible to do, as far as I can tell. If someone needs one (or more) of them and could submit a PR to get them added, that would be most welcome!
